src/app/components/core/developer/additional-services/additional-service-modal/additional-service-modal.component.html

```diff
@@ -119,13 +127,16 @@
 <app-confirmation-dialog id="dialog-delete-confirmation">
   <header class="modal-card-head">
     <p class="modal-card-title is-size-6">Delete</p>
-    <button type="button" class="delete" aria-label="close" (click)="closeDeleteModal('dialog-delete-confirmation')"></button>
+    <button type="button" class="delete" aria-label="close"
+      (click)="closeDeleteModal('dialog-delete-confirmation')"></button>
   </header>
   <section class="modal-card-body">
-    Deleting {{service?.name}} service can not be undone. Continue?
+    Deleting {{serviceName}} service can not be undone. Continue?
   </section>
   <footer class="modal-card-foot">
-    <button type="button" class="button is-small" (click)="closeDeleteModal('dialog-delete-confirmation')">Cancel</button>
-    <button type="button" class="button is-small is-danger" (click)="deleteService(service?.name)" [appDisableUntilResponse]="reenableButton">Delete</button>
+    <button type="button" class="button is-small"
+      (click)="closeDeleteModal('dialog-delete-confirmation')">Cancel</button>
+    <button type="button" class="button is-small is-danger" (click)="deleteService(serviceName)"
```

Comment:
These are the lines having meaningful changes, others are formatting fixes. We should have a lint config  file in GitHub.
service?.name vs serviceName